frigate/http.py

```diff
@@ -991,3 +992,19 @@ def ffprobe():
         )
 
     return jsonify(output)
+
+
+@bp.route("/vainfo", methods=["GET"])
+def vainfo():
+    vainfo = vainfo_hwaccel()
+    return jsonify(
+        {
+            "return_code": vainfo.returncode,
+            "stderr": vainfo.stderr.decode("unicode_escape").strip()
+            if vainfo.stderr.decode()
+            else {},
```

Comment:
Should this be an empty object or None/empty string?

Reply:
Empty string is probably better.